attractions/file-input/file-dropzone.svelte

```diff
@@ -106,11 +107,13 @@
     on:dragenter={() => dragActive = true}
     on:dragleave={() => { dragActive = false; wrongType = false; }}
     on:drop|preventDefault|stopPropagation={acceptUpload}
-    use:ripple={{ disabled }}
+    use:ripple={{ disabled: disabled || files.length >= max }}
```

Comment:
For some reason, the ripple is still present when I hit the limit of files

Reply:
This is because we do not react to changes in its parameters. The same would apply to, say, buttons if they are disabled after being previously enabled (though we wouldn't see that because the button itself is disabled, so no `click` event is fired).